launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -925,7 +925,18 @@ QString MinecraftInstance::getStatusbarDescription()
     if(m_settings->get("ShowGameTime").toBool())
     {
         if (lastTimePlayed() > 0) {
-            description.append(tr(", last played for %1").arg(Time::prettifyDuration(lastTimePlayed())));
+            struct tm * localTime_format;
+            localTime_format = localtime(lastLaunchTime());
+
+            char lastLaunchTime_formatted[13];
+            strftime(lastLaunchTime_formatted,13,"%Ex",localTime_format);
+
+            description.append(
+                tr(", last played at %1 for %2").arg(
+                    lastLaunchTime_formatted,
+                    Time::prettifyDuration(lastTimePlayed())
+                )
+            );
```

Comment:
Sadly `lastLaunchTime()` is not a method defined on the `MinecraftInstance` or `BaseInstance` classes

you instead was `lastLaunch()`.

Also, using `QDateTime` and `QLocale` will localize the date format

```suggestion
            QDateTime lastLaunchTime = QDateTime::fromMSecsSinceEpoch(lastLaunch());
            description.append(tr(", last played at %1 for %2")
                                   .arg(QLocale::system().toString(lastLaunchTime, QLocale::ShortFormat))
                                   .arg(Time::prettifyDuration(lastTimePlayed())));
```